ssg/build_remediations.py

```diff
@@ -60,7 +60,7 @@ def get_available_functions(build_dir):
         # ElementTree sorting XML attrs alphabetically. Hidden is guaranteed
         # to be the first attr and ID is guaranteed to be second.
         remediation_functions = re.findall(
-            r'<Value hidden=\"true\" id=\"function_(\S+)\"',
+            r'<Value.*id=\"function_(\S+)\"',
```

Comment:
I guess we can trust that XCCDF Values with id beginning with `function_` are remediation functions.

Can you also update the comment in the lines above?

Reply:
Yes, it is funny how things that look implementation-dependent eventually turn out to really be implementation-dependent :-)

Reply:
:octopus: